src/styles/utils/variables.scss

```diff
@@ -0,0 +1,2 @@
+$card-width: 200px;
+$trans-speed: 300ms;
```

Comment:
better to call it not the `trans-speed`, but `transition-duration` - it looks clearer this way

Reply:
Done! thank you very much Nazar!